Player gets kicked on final death

I wanted to test the Slimecican origin, and as i died the 3rd time i got kicked from the server for: "Internal server error" And all of my stuff is gone. When i log back in i´m in the death screen and can normally respawn. Also i dont bounce for some reason. And i dont get damage in hot biomes like the nether. Server console when i die:                                                                               
18.04 10:22:56 [Server] Server thread/INFO Player LC_Chicken chose Origin: slimeorigin:slimecican, for layer: origins:origin
18.04 10:23:25 [Server] Server thread/INFO There are 1 of a max of 20 players online: LC_Chicken
18.04 10:23:31 [Server] Server thread/WARN Failed to handle packet for /My IP
18.04 10:23:31 [Server] INFO net.minecraft.class_148: Ticking player
18.04 10:23:31 [Server] INFO at net.minecraft.class_3222.method_14226(class_3222.java:498) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_3244.method_18784(class_3244.java:219) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_2535.method_10754(class_2535.java:232) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_3242.method_14357(class_3242.java:173) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:882) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_3176.method_3813(class_3176.java:312) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:811) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:670) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_275]
18.04 10:23:31 [Server] INFO Caused by: java.lang.IllegalAccessError: tried to access field virtuoel.pehkui.api.PehkuiConfig.DATA from class net.minecraft.class_1309
18.04 10:23:31 [Server] INFO at net.minecraft.class_1309.revertScale(class_1309.java:6359) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_1309.handler$zjh000$tryUseTotem(class_1309.java:6424) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_1309.method_6095(class_1309.java) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_1309.method_5643(class_1309.java:1101) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_1657.method_5643(class_1657.java:914) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_3222.method_5643(class_3222.java:631) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_1309.method_5670(class_1309.java:361) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_1297.method_5773(class_1297.java:431) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_1309.method_5773(class_1309.java:2209) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_1657.method_5773(class_1657.java:274) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO at net.minecraft.class_3222.method_14226(class_3222.java:434) ~[intermediary-server.jar:?]
18.04 10:23:31 [Server] INFO ... 9 more
18.04 10:23:31 [Disconnect] User LC_Chicken has disconnected, reason: Internal server error
18.04 10:23:31 [Server] Server thread/INFO LC_Chicken left the game

My mods: extraorigins, fabric api, lithium, moborigins, Origins, pehkui, phosphor, slimeorigin.

--- a/src/main/java/dev/jacksibley/slimeorigin/mixin/MixinLivingEntity.java
+++ b/src/main/java/dev/jacksibley/slimeorigin/mixin/MixinLivingEntity.java
@@ -1,8 +1,5 @@
 package dev.jacksibley.slimeorigin.mixin;
 
-
-import com.google.gson.JsonElement;
-import com.google.gson.JsonPrimitive;
 import dev.jacksibley.slimeorigin.Slimeorigin;
 import net.minecraft.entity.Entity;
 import net.minecraft.entity.EntityType;
@@ -68,10 +65,8 @@ public abstract class MixinLivingEntity extends Entity {
 
     private void revertScale()
     {
-        if (Optional.ofNullable(PehkuiConfig.DATA.get("keepScaleOnRespawn"))
-            .filter(JsonElement::isJsonPrimitive).map(JsonElement::getAsJsonPrimitive)
-            .filter(JsonPrimitive::isBoolean).map(JsonPrimitive::getAsBoolean)
-            .orElse(false)) {
+        PehkuiConfig pehconf = new PehkuiConfig();
+        if ((Boolean)Optional.ofNullable((Boolean)PehkuiConfig.COMMON.keepAllScalesOnRespawn.get()).orElse(false)) {
                 int flooredSlime = (int) floor(this.attributes.getValue(SLIME_SIZE));
                 if (flooredSlime != 0)
                 {

--- a/src/main/resources/data/slimeorigin/powers/fragmentation.json
+++ b/src/main/resources/data/slimeorigin/powers/fragmentation.json
@@ -10,6 +10,11 @@
     "command": "attribute @s slimeorigin:slime_size base set 0",
     "permission_level": 4
   },
+  "entity_action_respawned": {
+    "type": "origins:execute_command",
+    "command": "attribute @s slimeorigin:slime_size base set 3",
+    "permission_level": 4
+  },
   "execute_chosen_when_orb": true,
   "name": "Fragmentation",
   "description": "When others would die, you shrink to half your size. (Twice per life)"

